### PR TITLE
Dm user with reasons for their mute/ban

### PR DESCRIPTION
**Pull request by Simyon264**
Ye

### CODE REVIEW

File: commands/ban.js [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
+const discord = require('discord.js');
 
 var colourInfo = config.messageColours.info;
 var colourWarn = config.messageColours.warn;
 
 exports.run = function(message, prefix, args) {
 	if (message.member.roles.find(role => role.name == config.roles.moderator)) {
 	    var user = message.mentions.users.first();
-	
+	    var name = message.guild.name;
```
Comment (Vylpes) [inline commands/ban.js:11]: `name` is ambiguous. Variable names should be clearly named as to exactly what it is. Something like `serverName` would be a better option

File: commands/kick.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+const discord = require(`discord.js`)
 
 var colourInfo = config.messageColours.info;
 var colourWarn = config.messageColours.warn;
 
 exports.run = function(message, prefix, args) {
 	if (message.member.roles.find(role => role.name == config.roles.moderator)) {
-	    var user = message.mentions.users.first();
+		var user = message.mentions.users.first();
+		var name = message.guild.name;
```
Comment (Vylpes) [inline commands/kick.js:11]: Same as above

File: commands/mute.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+const discord = require('discord.js');
 
 var colourInfo = config.messageColours.info;
 var colourWarn = config.messageColours.warn;
 
 exports.run = function(message, prefix, args) {
 	if (message.member.roles.find(role => role.name == config.roles.moderator)) {
-	    var user = message.mentions.users.first();
+		var user = message.mentions.users.first();
+		var name = message.guild.name;
```
Comment (Vylpes) [inline commands/mute.js:11]: and here too

Comment (Simyon264) [reply commands/mute.js:11]: :(

File: commands/ban.js (hunk 1 above)
Comment (Simyon264) [reply commands/ban.js]: Changed

File: commands/kick.js (hunk 2 above)
Comment (Simyon264) [reply commands/kick.js]: Changed

File: commands/mute.js (hunk 3 above)
Comment (Simyon264) [reply commands/mute.js]: Changed

**Pull request merged**